agora/Agora/Effect/TreasuryWithdrawal.hs

```diff
@@ -127,71 +128,72 @@ instance PTryFrom PData PTreasuryWithdrawalDatum
 
      @since 0.1.0
```

Comment:
Here, and anywhere where the interface has changed, bump the version to 1.0.0. (Too many instances to mention them all.)